build/yamls/base/controller-rbac.yml

```diff
@@ -110,6 +119,17 @@ rules:
       - get
       - watch
       - list
+  - apiGroups:
+      - security.antrea.tanzu.vmware.com
+    resources:
+      - tiers
+    verbs:
+      - get
+      - watch
+      - list
+      - create
+      - patch
+      - update
```

Comment:
I may have missed something because Github is collapsing some of the large files, but do you need all of these? In particular, I don't know about `patch` and `update`?

Reply:
removed `patch` and `update`.